governance-aepp/src/views/Account.vue

```diff
@@ -270,15 +281,48 @@
   }
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
```

Comment:
tailwind style css over custom classes

Reply:
tailwind style misses the required sizes like 15px font or 10px paddings

Reply:
alternatively a tailwind config to change the class sizes could be used, tough might have unexpected results and additional refactoring.